sentry-android-core/src/main/java/io/sentry/android/core/ContextUtils.java

```diff
@@ -346,6 +351,29 @@ static boolean isForegroundImportance(final @NotNull Context context) {
     }
   }
 
+  /** Register a not exported BroadcastReceiver, independently from platform version. */
+  static @Nullable Intent registerReceiver(
+      final @NotNull Context context,
+      final @NotNull SentryOptions options,
+      final @Nullable BroadcastReceiver receiver,
+      final @NotNull IntentFilter filter) {
+    return registerReceiver(context, new BuildInfoProvider(options.getLogger()), receiver, filter);
+  }
+
+  /** Register a not exported BroadcastReceiver, independently from platform version. */
+  @SuppressLint({"NewApi", "UnspecifiedRegisterReceiverFlag"})
+  static @Nullable Intent registerReceiver(
+      final @NotNull Context context,
+      final @NotNull BuildInfoProvider buildInfoProvider,
+      final @Nullable BroadcastReceiver receiver,
+      final @NotNull IntentFilter filter) {
+    if (buildInfoProvider.getSdkInfoVersion() >= Build.VERSION_CODES.TIRAMISU) {
+      return context.registerReceiver(receiver, filter, RECEIVER_NOT_EXPORTED);
```

Comment:
`h`: hm, as I understood we should set it to `RECEIVER_EXPORTED` if we're listening for any system events. From https://developer.android.com/guide/components/broadcasts#context-registered-receivers:

>  If this receiver is listening for broadcasts sent from the system or from other apps—even other apps that you own—use the RECEIVER_EXPORTED flag

Reply:
yes, exactly 😅